### PR TITLE
Pay reward back to miners when don't have znode

### PR DESCRIPTION
**Pull request by nopslide**
Reward should belong to the miner who are creating a block during network have zero znodes enabled

### CODE REVIEW

File: src/miner.cpp [hunk 1]
```diff
@@ -529,8 +529,10 @@ CBlockTemplate* BlockAssembler::CreateNewBlock(
         if (nHeight >= chainparams.GetConsensus().nZnodePaymentsStartBlock) {
             const Consensus::Params &params = chainparams.GetConsensus();
             CAmount znodePayment = GetZnodePayment(chainparams.GetConsensus(), nHeight > 0 && nBlockTime >= params.nMTPSwitchTime);
-            coinbaseTx.vout[0].nValue -= znodePayment;
-            FillBlockPayments(coinbaseTx, nHeight, znodePayment, pblock->txoutZnode, pblock->voutSuperblock);
+            if(mnodeman.CountEnabled() > 0){
```
Comment (martun) [inline src/miner.cpp]: I'm not familiar with znodes. Can it happen, that mnodeman.CountEnabled() > 0, but they did not agree on the znode winner, and we still want to pay it to miner?

Comment (nopslide) [reply src/miner.cpp]: it checked either we have znodes running on network or not

**Comment by reubenyap** [comment]
Not to touch this until Znode revamp.

**Comment by nopslide** [comment]
it should not be close, but open until we release sigma and look back at it later.

**Comment by qashban4** [comment]
As per project meeting discussion on 23-09-2019: This issue is irrelevant as we are upgrading to core 0.14, so this issue will be closed.

**Pull request closed without merging**